float64[7] tau_error
float64[7] tau_commanded
float64[7] tau_measured
float64 root_mean_square_error
